src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -292,20 +292,11 @@ internal void StartInternal()
             );
             grainInterfaceMap = transport.GetTypeCodeMap(grainFactory).Result;
 
-            if (!string.IsNullOrEmpty(grainInterfaceMap.ClusterId))
-            {
-                clientId = GrainId.NewClientId(handshakeClientId.PrimaryKey, grainInterfaceMap.ClusterId);
-                transport.UpdateClientId(clientId);
-                CurrentActivationAddress = ActivationAddress.GetAddress(transport.MyAddress, clientId, CurrentActivationAddress.Activation);
-            }
-            else
-            {
-                clientId = handshakeClientId;
-            }
-
             StreamingInitialize();
         }
 
+        bool firstMessageReceived;
```

Comment:
The general coding style is to declare fields and properties at the top of their class definition. Could you move it up.